### ISSUE TITLE
Missing nserver entry in local-config.cfg (as in dockerfile)

### ISSUE THREAD

**Issue by danfe** [Stiffstream/arataga#4]
Looks like `local-config.cfg` file example, as created in e.g. dockerfile, lacks `nserver` line which prevents the program from initialising properly and start listening on a port.  There is a complaint in the log, but it's merely a warning:
```
[2021-05-28 10:19:03.819] [arataga] [warning] config_processor: load local config file at startup failed: config_parser: At least one name server IP should be specified
```
Only after I've added a nameserver, it started to listen on 5001 port and process requests.

**Comment by eao197** [comment]
Thanks! The fix is in the master branch now.

**Comment by danfe** [comment]
Fixed by 42a830d, closing.
